corehq/apps/app_manager/static/app_manager/js/download_async_modal.js

```diff
@@ -99,6 +99,7 @@ hqDefine('app_manager/js/download_async_modal', function () {
             downloader = asyncDownloader($modal);
         downloader.generateDownload($modal.data("url"));
     };
+    $(document).on('click','.download-zip',downloadApplicationZip);
```

Comment:
This module doesn't need to return `downloadApplicationZip` anymore.

Reply:
Can you update the docs [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/download_async_modal.html) to reflect the new style?